inbox/models/message.py

```diff
@@ -265,7 +273,7 @@ def create_from_synced(cls, account, mid, folder_name, received_date,
             # Non-persisted instance attribute used by EAS.
             msg.parsed_body = parsed
             msg._parse_metadata(parsed, body_string, received_date, account.id,
-                                folder_name, mid)
+                                account.namespace.public_id, folder_name, mid)
```

Comment:
what is account.namespace.public_id here? isn't it the same as account.id?

Reply:
No, they are not the same. What we see in communication between Nylas and node.js applications (dexter, orchestra) - is actually namespace public id. And it's used through all the Nylas. So this is correct.

Reply:
It has one-to-one relation with Account table.